core/domain/role_services_test.py

```diff
@@ -51,3 +52,11 @@ def test_action_allocated_to_all_allowed_roles(self) -> None:
 
         self.assertItemsEqual(  # type: ignore[no-untyped-call]
             list(role_actions), feconf.ALLOWED_USER_ROLES)
+
+    def test_log_role_query(self) -> None:
+        role_services.log_role_query(
+            'TEST_USER', feconf.ROLE_ACTION_ADD, role='GUEST')
+        self.assertEqual(
```

Comment:
There should be a pre-check to make sure this was false before.

See guidance at https://github.com/oppia/oppia/wiki/Backend-tests#guidelines-for-writing-good-tests (point 2).

Reply:
Gotcha, I'll add the precheck. Thank you!